RadarSDK/RadarVerificationManager.m

```diff
@@ -194,6 +198,11 @@ - (void)startTrackingVerifiedWithInterval:(NSTimeInterval)interval beacons:(BOOL
                 minInterval = minInterval - 10;
             }
             
+            // min interval is 10 seconds
+            if (minInterval < 10) {
+                minInterval = 10;
+            }
+            
```

Comment:
Not really an issue with the new code here, but `startTrackingVerifiedWithInterval` initiates a `trackVerified` immediately (unlike the other `startTracking`) — we don't have a check on `self.started` or `sending`. We maybe want one to prevent simultaneous `trackVerified`s.

At the beginning of `startTrackingVerifiedWithInterval`:
```
if (self.started) {
    [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:@"Already tracking verified"];
    return
}
```

Reply:
Independent of `startTracking()`, the desired behavior here is definitely to do a track right away. I've tested this extensively in simulator / on real device and haven't observed duplicate tracks, so I don't think we need to make this change right now

Reply:
Cool, sounds good. Just a way to make things more robust to imperfect implementations (multiple `startTrackingVerified` calls). Agree that it's not necessary to change atm